graphdatascience/tests/integration/test_coverage.py

```diff
@@ -59,8 +59,6 @@
     "gds.alpha.create.cypherdb",
     "gds.alpha.backup",  # TODO: Add support for this
     "gds.alpha.restore",  # TODO: Add support for this
-    "gds.alpha.config.limits.set",  # TODO: Add support for this
-    "gds.alpha.config.limits.list",  # TODO: Add support for this
```

Comment:
Originally, I wanted to remove these with #203 but the functionality does not work well with authentication turned off.

Reply:
The coverage doesn't call anything, so should not be a problem? It just checks if there are callable Python objects

Reply:
The tests are green it seems